terracumber-cli

```diff
@@ -187,7 +187,9 @@ def run_terraform(args, tf_vars):
                                                     args.logfile, args.terraform_bin)
     if args.custom_repositories:
         with open(args.custom_repositories, 'r') as repos_file:
-            terraform.inject_repos(repos_file)
+            if not terraform.inject_repos(repos_file):
+                print("ERROR: could not inject custom repositories into file %s" % args.tf)
```

Comment:
We should tell the user why.

IIUC the code, this happens when you specified `--custom-repositories` but there are 0 or more than 1 place holders?

Reply:
Yes, but the return value `False` is generic. What happens in the other file is unknown here.

Reply:
I'll add another message in the other file.

Reply:
Why? I only see a return False at the function if there are 0 or more than 1 placeholders, and True otherwise.... and that reminds me that I need to add other comment there.

Reply:
Why? Limited responsability of each component. Only information conveyed by `False` is that it failed, no explanation why.

Anyway. I added another commit with another message. I don't like it though, as you seem to have all error management in the terracumber-cli file, and this is breaking it.

Reply:
I could of course break the separation of responsibilities and put the message about missing placeholder in the cli file. Tell me if that's what you want to do.

Reply:
i did that. Is it better now?

Reply:
This part yes, but see next comment I am going to leave.